stingray/lightcurve.py

```diff
@@ -1334,21 +1334,7 @@ def estimate_segment_size(self, min_total_counts=100, min_time_bins=100):
         >>> min_total_bins = 40
         >>> assert np.isclose(lc.estimate_segment_size(100, 40), 8.0)
         """
-
-        rough_estimate = np.ceil(min_total_counts / self.meancounts) * self.dt
-
-        segment_size = np.max([rough_estimate, min_time_bins * self.dt])
-
-        keep_searching = True
-        while keep_searching:
-            start_times, stop_times, results = self.analyze_lc_chunks(segment_size, np.sum)
-            mincounts = np.min(results)
-            if mincounts >= min_total_counts:
-                keep_searching = False
-            else:
-                segment_size += self.dt
-
-        return segment_size
+        return super().estimate_segment_size(min_total_counts, min_time_bins)
 
     def analyze_lc_chunks(self, segment_size, func, fraction_step=1, **kwargs):
```

Comment:
I like `analyze_segments` better than `analyze_lc_chunks`. Do you think it would be worth deprecating this method and have it simply use the superclass method here?

Reply:
Done